Server/AIServer/ServerDlg.cpp

```diff
@@ -824,18 +843,18 @@ BOOL CServerDlg::LoadNpcPosTable(std::vector<model::NpcPos*>& rows)
 						const std::string& path = *row->Path;
 						if ((row->PathPointCount * CharactersPerPoint) > path.size())
 						{
-							CString error;
-							error.Format(
-								_T("LoadNpcPosTable: NPC expects a larger path for this DotCnt[%d] (path point count) - zone=%d nid=%d, sid=%d, name=%hs, acttype=%d, path=%d #####\n"),
+							std::string error = std::format("LoadNpcPosTable: NPC expects a larger path for this PathPointCount [zoneId={} serial={} npcId={} npcName={} moveType={}, pathCount={}]",
 								row->ZoneId,
 								row->PathPointCount,
 								pNpc->m_sNid + NPC_BAND,
 								pNpc->m_sSid,
 								pNpc->m_strName.c_str(),
 								pNpc->m_byMoveType,
 								pNpc->m_sMaxPathCount);
-							TRACE(error);
-							AfxMessageBox(error);
+							spdlog::error(error);
+							std::wstring_convert<std::codecvt_utf8<wchar_t>> converter;
```

Comment:
We have existing wrapper code for "this", though it's Windows-centric right now, but it can be swapped out later as needed.
See: shared\StringConversion.h

*Technically* right now it should be CpToWide(error, 949) for Korean, but LocalToWide() is sufficient. The names are probably more likely to be encoded using their local encoding right now instead of utf8, and I suspect any Korean is just mangled in the DB.